benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -268,11 +280,21 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
     }
   }
 
-  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoFileDescriptorSupplier {
+  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoDescriptorSupplier {
     @java.lang.Override
     public com.google.protobuf.Descriptors.FileDescriptor getFileDescriptor() {
       return io.grpc.benchmarks.proto.Services.getDescriptor();
     }
+
+    @java.lang.Override
+    public com.google.protobuf.Descriptors.ServiceDescriptor getServiceDescriptor() {
+      return getFileDescriptor().getServices().get(0);
```

Comment:
The index is hardcoded but it's not static and same for all. It uses the index of the service in the file, see https://github.com/grpc/grpc-java/pull/2904/files#diff-4e543a2f8a0243d4e0fed21737cdbf18R926